source/wp-content/themes/wporg-developer/scss/main.scss

```diff
@@ -2388,6 +2388,7 @@ body.responsive-show {
 
 	#page {
 		overflow-x: hidden;
```

Comment:
Since `overflow-x: clip` is not supported on Safari 15.5, 15.4. ([detail](https://caniuse.com/?search=overflow%3A%20clip)), and it's [suggested](https://github.com/WordPress/wporg-developer/pull/83#discussion_r891272205) to support them, so the attribute is retained.